flecsi/data/topology_accessor.hh

```diff
@@ -40,11 +40,11 @@ namespace data {
  */
 template<class T, Privileges Priv>
 struct topology_accessor
-  : T::template interface<typename T::core::template access<Priv>>,
+  : T::template interface_<typename T::core::template access<Priv>>,
```

Comment:
This change got reverted, right?  If you don't mind, I'd like it excised from the history.

Reply:
I already have done so. This will disappear with the next push to this branch.